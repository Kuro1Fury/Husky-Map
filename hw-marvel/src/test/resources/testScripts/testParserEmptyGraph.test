# Test the parser on an empty graph

LoadGraph chars empty.csv
