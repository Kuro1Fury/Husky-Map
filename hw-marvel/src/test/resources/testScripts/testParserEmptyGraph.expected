# Test the parser on an empty graph

loaded graph chars